crates/storage/nippy-jar/src/lib.rs

```diff
@@ -271,6 +271,21 @@ impl<H: NippyJarHeader> NippyJar<H> {
         self.path.with_extension(CONFIG_FILE_EXTENSION)
     }
 
+    /// Deletes from disk this [`NippyJar`] alongside every satellite file.
+    pub fn delete(self) -> Result<(), NippyJarError> {
+        // TODO(joshie): ensure consistency on unexpected shutdown
```

Comment:
this todo supposed to be here still?

Reply:
yes, created the associated issue: 
https://github.com/paradigmxyz/reth/issues/6187